bin/receiver.py

```diff
@@ -37,35 +37,41 @@ def main():
     ver = "SSM %s.%s.%s" % __version__
     default_conf_location = '/etc/apel/receiver.cfg'
     default_dns_location = '/etc/apel/dns'
-    op = OptionParser(description=__doc__, version=ver)
-    op.add_option('-c', '--config',
-                  help=('location of config file, '
-                        'default path: ' + default_conf_location),
-                  default=default_conf_location)
-    op.add_option('-l', '--log_config',
-                  help='DEPRECATED - location of logging config file (optional)',
-                  default=None)
-    op.add_option('-d', '--dn_file',
-                  help=('location of the file containing valid DNs, '
-                        'default path: ' + default_dns_location),
-                  default=default_dns_location)
-
-    options, unused_args = op.parse_args()
+    arg_parser = ArgumentParser(description=__doc__)
+
+    arg_parser.add_argument('-c', '--config',
+                            help='location of config file, default path: '
+                                  '%s' % default_conf_location,
+                            default=default_conf_location)
+    arg_parser.add_argument('-l', '--log_config',
+                            help='DEPRECATED - location of logging config '
+                                 'file (optional)',
+                            default=None)
+    arg_parser.add_argument('-d', '--dn_file',
+                            help='location of the file containing valid DNs, '
+                                  'default path: %s' % default_dns_location,
+                            default=default_dns_location)
+    arg_parser.add_argument('-v', '--version',
+                            action='version',
+                            version='ver: %s' % ver)
```

Comment:
```suggestion
                            version=ver)
```

Sorry, I wasn't clear. I meant literally just use `ver` as the value as it's already a string with all the info we need. Same for the other file.